src/index.test.ts

```diff
@@ -1,7 +1,9 @@
 import { deepEqual } from "node:assert/strict";
 import { test } from "node:test";
 
-function greet() {}
+function greet(name:string): object {
```

Comment:
`object` as a return type is almost never something we use.
You could use `{message: string}` as the return type instead.
I would also recommend declaring a type to make the code easier to read: `type Greeting = {message:string}`.